XRTK-Core/Packages/com.xrtk.core/Editor/Utilities/UnityFileHelper.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) XRTK. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections.Generic;
+using System.IO;
+
+namespace XRTK.Editor.Utilities
+{
+    public static class UnityFileHelper
+    {
+        private static readonly string[] UnityFileExtensions = {
+            "*.meta",
+            "*.mat",
+            "*.anim",
+            "*.prefab",
+            "*.unity",
+            "*.asset",
+            "*.guiskin",
+            "*.fontsettings",
+            "*.controller",
+        };
+
+        public static List<string> GetUnityAssetsAtPath(string assetsRootPath)
```

Comment:
Can we add some documentation here?

I'd like to explicitly say that we're returning full paths

Reply:
Done